packages/schematics/src/collection/cypress-project/index.ts

```diff
@@ -73,6 +73,8 @@ function checkDependenciesInstalled(): Rule {
     const dependencyList: { name: string; version: string }[] = [];
     if (!packageJson.devDependencies.cypress) {
       dependencyList.push({ name: 'cypress', version: cypressVersion });
+      // NOTE: Need to be removed on the next Cypress release (=>3.1.1)
+      dependencyList.push({ name: '@types/jquery', version: '3.3.6' });
```

Comment:
This code relates to this issue https://github.com/cypress-io/cypress/issues/2363. The PR has been merged, but we are waiting for a new Cypress release.